forum/qa-plugin/report-reason/script.js

```diff
@@ -0,0 +1,95 @@
+$(document).ready(function()
+{
+    $(".qa-form-light-button-flag").attr("type", "button");
+    
+    $(".qa-form-light-button-flag").click( function()
```

Comment:
To i linijka 3 (oraz każdy pozostały przypadek) - elementy z DOM pobieramy raz, potem działamy na referencjach do nich w taki sposób żeby nie musieć więcej razy odnosić się do *document*.